src/backend/partaj/core/api/user_lite.py

```diff
@@ -0,0 +1,61 @@
+"""
+User lite related API endpoints.
+"""
+from django.contrib.auth import get_user_model
+
+from rest_framework import viewsets
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+
+from ..forms import UserListQueryForm
+from ..indexers import ES_CLIENT, UsersIndexer
+from .common import AutocompleteMixin, ViewSetMetadata
+
+User = get_user_model()
+
+
+class UserLiteViewSet(AutocompleteMixin, viewsets.GenericViewSet):
+    """
+    API endpoints for lite users.
+    Uses ElasticSearch to always return lists of lite users, accessible to all
+    authenticated users.
+    """
+
+    _meta = ViewSetMetadata(indexer=UsersIndexer)
+
+    permission_classes = [IsAuthenticated]
+
+    def list(self, request, *args, **kwargs):
```

Comment:
As this method is quite the same for topic, user and unit, we should factorize it one day. Maybe add a comment to remember?

Reply:
I think it's similar now (only the sorting changes, and could be passed as a key from the client side) but it's the kind of function which would diverge over time. If you look at the one for referral lite it is different.

The analogous fields don't have the same name (eg. first name + last name for user VS name for topic) and the kinds of search we'll run can be different and run on different fields.

If it does end up staying very similar as we build more use cases for these endpoints factorizing them would definitely make sense though 👍